### ISSUE TITLE
when subscriptions triggered by pg_notify, permission denied but no error tips

### ISSUE THREAD

**Issue by surmind** [graphile/postgraphile#1017]
I'm submitting a ... 

* [ ] bug report
* [x] feature request
* [ ] question 

PostGraphile version: 4.4.0-beta.1

It seems whatever role I used to do subscriptions, when pg_notify been called, postgrahile use default role to do the query. For example, when set `pgDefaultRole` to `anonymous`, but login and do subscription as `admin`, postgraphil still use `anonymous` to query the relatedNode. and if `anonymous` don't have permission to do the query, subscriber get null instead of error tips.

![image](https://user-images.githubusercontent.com/679318/54025911-fd4d0200-41d6-11e9-806e-d20338d1938f.png)

I think it's better to report an error in query result or at least do some explain in document, so a postgraphile newbie like me don't need to doubt self when follow the doc but can't get data as expected.
  
After that, does this mean if I want to do subscriptions, I must start another postgraphile instance which use a role have permission as default role, then limit request follow [subscription-security](https://www.graphile.org/postgraphile/subscriptions/#subscription-security) ?

**Comment by benjie** [comment]
I'm not sure if this is a bug or misconfiguration. Let's dig in...

Two questions to get us started:

1. How is your authentication configured - using JWT or `pgSettings`?
2. How are you sending that authentication information to the server along with the websocket request? (If you can include two screenshots of the Chrome Developer Tools' Network Panel, one showing the request headers and the other showing the initial frames for the websocket request that would be extremely helpful: https://www.graphile.org/postgraphile/debugging/#step-1-check-youre-requesting-what-you-think-youre-requesting )

**Comment by surmind** [comment]
Thank you so much, I found the reason, I don't send `authorization` in websocket.
I use JWT for authorization, but add it use a chrome plugin called ModHeader, which only add header in http but not in websocket.
After upgrade to 4.4.0, I saw there is a header panel, but still use the plugin for add header.

Now, after use header panel for authorization, the token pass through websocket and get result correct. Thank you.
